rllib/algorithms/sac/torch/sac_torch_rl_module.py

```diff
@@ -0,0 +1,134 @@
+from typing import Any, Dict, List, Tuple
+
+from ray.rllib.algorithms.sac.sac_rl_module import QF_PREDS
+from ray.rllib.algorithms.sac.sac_rl_module import SACRLModule
+from ray.rllib.core.models.base import ENCODER_OUT
+from ray.rllib.core.rl_module.torch.torch_rl_module import TorchRLModule
+from ray.rllib.core.rl_module.rl_module import RLModule
+from ray.rllib.core.rl_module.rl_module_with_target_networks_interface import (
+    RLModuleWithTargetNetworksInterface,
+)
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.annotations import override
+from ray.rllib.utils.framework import try_import_torch
+from ray.rllib.utils.nested_dict import NestedDict
+from ray.rllib.utils.typing import NetworkType
+
+
+torch, nn = try_import_torch()
+
+
+class SACTorchRLModule(TorchRLModule, SACRLModule):
+    framework: str = "torch"
+
+    @override(RLModule)
+    def _forward_inference(self, batch: NestedDict) -> Dict[str, Any]:
+        output = {}
```

Comment:
Awesome! I love how simple the inference/exploration forward passes have become. They used to be such a mess in the old stack.